npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -436,6 +450,15 @@ def run_npm_checker(path:, version:)
           end
         end
 
+        def run_npm7_checker(version:)
+          SharedHelpers.run_shell_command(
+            "npm install #{dependency.name}@#{version} --package-lock-only --ignore-scripts"
```

Comment:
I wonder if we need to run `--dry-run=false`? it will be set to `true` in the updater container, I've set it before on my machine or in the container and re-run to see if tests still pass, e.g. `npm config set dry-run true`

Reply:
I'd previously set it to `dry-run=true` explicitly and that worked fine to check peer deps, should we add to for clarity?

Reply:
Ah right, yeah think it would be a good idea to add it explicitly as I imagine we'll drop the npm config in the updater at some point when we deprecate npm 6